client/src/api/okteto.ts

```diff
@@ -100,10 +101,28 @@ const up = (manifestFile: string, contextName: string, onOutputChange: (stdout:
   });
 };
 
+const status = async ( manifest: string, contextName: string): Promise<OktetoStatus> => {
+  try {
+    const args = ['status', '-f', manifest, '-c', contextName];
```

Comment:
We should make sure that we receive always a json output. We should pass the `--log-output=json` parameter:
```suggestion
    const args = ['status', '-f', manifest, '-c', contextName, '--log-output', 'json'];
```

Please, also update tests to take this into account.